imports/ui/JapaneseNewGame.html

```diff
@@ -301,7 +301,7 @@
 
 <template name="jpn_points">
 	<span class="player-select-header"><b>Points: </b></span>
-	<select name="points">
+	<select name="points" class="points">
```

Comment:
Name is used as a reference for javascript whereas class is meant to be used by css. IDK if they can be merged, but we might be able to remove name if it's unused. For now, I'd want to leave it just in case.